packages/account-v2/src/containers/ProofOfIncome/ProofOfIncomeForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { StandaloneArrowUpFromBracketBoldIcon } from '@deriv/quill-icons';
+import { Button, Divider, Text, useDevice } from '@deriv-com/ui';
+import { FormDropDownField } from '../../components/FormFields';
+import { FormDocumentUploadField } from '../../components/FormFields/FormDocumentUploadField';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { ProofOfIncomeUtils } from '../../utils/proofOfIncomeUtils';
+
+export const ProofOfIncomeForm = () => {
+    const PoincDocumentsList = ProofOfIncomeUtils();
+    const { isMobile } = useDevice();
+    const listItems = [
+        'The document must be up-to-date and signed by the issuance authority.',
+        'The document must contain a letterhead.',
+        'Invalid or incomplete documents shall be rejected.',
+    ];
+    const documentValidation = Yup.mixed().required('Please upload a file');
```

Comment:
To be explicit that only File should be uploaded, u can typecast the mixed function as `Yup.mixed<File>()`

Reply:
![Screenshot 2024-04-26 at 3 30 38 PM](https://github.com/binary-com/deriv-app/assets/158162395/7ac79a1e-3db2-4ec7-a824-7c1c0e43ba93)

something like this ?